installing/disconnected_install/installing-mirroring-disconnected-v2.adoc

```diff
@@ -0,0 +1,215 @@
+----
+
+More Image Set Configuration example can be found in the additional resources.
+
+[id="oc-mirror-workflows"]
+== oc-mirror workflows
+
+This section will explain all the workflows supported by oc-mirror currently.
+
+.Mirror to Disk
+
+Pulls the container images from the source specified in the image set configuration and pack them into a tar archive on disk (local directory).
+
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml <file path> --v2
+----
+
+.Disk to Mirror
+
+Copy the containers images from the tar archive to a container registry (--from flag is required on this workflow).
+
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml --from file:///oc-mirror/mirror1 docker://localhost:6000 --v2
+----
+
+.Mirror To Mirror
+
+Copy the container images from the source specified in the image set configuration to the destination (container registry).
+
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml --workspace ffile:///oc-mirror/mirror1 docker://localhost:6000 --v2
+----
+
+[id="oc-mirror-delete"]
+== Image delete workflow in oc-mirror
+
+In V1, pruning (or deletion of images) is set automatically, if a specific index (release, catalog) or additionalImage is to be deleted, the current workflow would be to use the same "StorageConfig" setting and omit the previously mirrored images, the pruning logic will find the "difference" and prune the images accordingly.
+
+This has its pros and cons, but due to the large amount of bugs been filed, we decided to change the behaviour in v2.
+
+In v2 a new kind called "DeleteImageSetConfiguration" has been introduced, this is used to configure the deletion of images, to avoid using an ImageSetConfig, accidentally deleting wanted (deployed) images.
+
+A new oc-mirror delete sub command is now available to execute the delete workflow with the DeleteImageSetConfiguration input.
+
+This functionality is able to work in a complete "air-gapped" (disconnected) environment, provided that the images have been previously mirrored (i.e the local cache and remote registry know about the images being deleted).
+
+The delete functionality is split into 2 stages:
+
+- The first stage is to create (generate) a delete yaml file using the --generate flag, this file is used to validate the images/blobs that will be deleted.
+- The second stage is to use the delete yaml file (once validated) to actually perform a local cache and remote registry delete.
+
+.For stage 1:
+** Use the following command to create `delete` yaml file for delete phase:
+
+[source,terminal]
+----
+$ oc-mirror delete --config delete-image-set-config.yaml --workspace file://<previously-mirrored-work-folder> --v2 --generate docker://<remote-registry> 
+----
+
+** Use the following command to compare versions:
+
+[source,terminal]
+----
+$ oc-mirror delete --config delete-image-set-config.yaml --workspace file://<previously-mirrored-work-folder> --v2 --generate --delete-id v4.11 docker://<remote-registry>
+----
```

Comment:
It is not very clear on why some one needs to to this. It would be great if you could explain on why we are doing this

Reply:
Yes, as discussed I will be adding the necessary information.